### PR TITLE
fdupes: add delete and noprompt examples, add French translation

### PR DESCRIPTION
**Pull request by angauber**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Looking for duplicates is great, being able to get rid of them is event better :sweat_smile:

### CODE REVIEW

File: pages.fr/common/fdupes.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# fdupes
+
+> Trouve les fichiers dupliqués dans les dossiers donnés.
+> Plus d'informations: <https://github.com/adrianlopezroche/fdupes>.
```
Comment (bl-ue) [inline pages.fr/common/fdupes.md:4]: Missing space?
```suggestion
> Plus d'informations : <https://github.com/adrianlopezroche/fdupes>.
```

File: pages.fr/common/fdupes.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Cherchez dans un dossier:
```
Comment (bl-ue) [inline pages.fr/common/fdupes.md:6]:
```suggestion
- Cherchez dans un dossier :
```

File: pages.fr/common/fdupes.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`fdupes {{dossier}}`
+
+- Cherchez dans plusieurs dossiers:
```
Comment (bl-ue) [inline pages.fr/common/fdupes.md:10]:
```suggestion
- Cherchez dans plusieurs dossiers :
```

File: pages.fr/common/fdupes.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`fdupes {{dossier1}} {{dossier2}}`
+
+- Cherchez dans un dossier recursivement:
```
Comment (bl-ue) [inline pages.fr/common/fdupes.md:14]:
```suggestion
- Cherchez dans un dossier recursivement :
```

File: pages.fr/common/fdupes.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`fdupes -r {{dossier}}`
+
+- Cherchez dans plusieurs dossiers dont un recursivement:
```
Comment (bl-ue) [inline pages.fr/common/fdupes.md:18]:
```suggestion
- Cherchez dans plusieurs dossiers dont un recursivement :
```

File: pages.fr/common/fdupes.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`fdupes {{dossier1}} -R {{dossier2}}`
+
+- Cherche recursivement et demande le fichier a conserver, supprimant les autres dupliqués
```
Comment (bl-ue) [inline pages.fr/common/fdupes.md:22]: Missing space and colon?
```suggestion
- Cherche recursivement et demande le fichier a conserver, supprimant les autres dupliqués :
```

File: pages.fr/common/fdupes.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`fdupes -rd {{dossier}}`
+
+- Cherche recursivement et supprime les dupliqués automatiquement
```
Comment (bl-ue) [inline pages.fr/common/fdupes.md:26]:
```suggestion
- Cherche recursivement et supprime les dupliqués automatiquement :
```

File: pages/common/fdupes.md [hunk 8]
```diff
@@ -11,10 +11,18 @@
...
-- Search all directories recursively:
+- Search a directory recursively:
 
 `fdupes -r {{directory}}`
 
 - Search multiple directories, one recursively:
 
 `fdupes {{directory1}} -R {{directory2}}`
+
+- Search recursively and prompt for file to preserve, deleting all other duplicates:
```
Comment (bl-ue) [inline pages/common/fdupes.md:22]: The current sentence doesn't read well.  Does this seem right?
```suggestion
- Search recursively for duplicates and display interactive prompt to pick which ones to keep, deleting the others:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hi @angauber and thank you for the new page!  I've left some comments below for you to review.

Note: the spaces I've recommened in the French sentences are the thin unbreakable space `0xa0`, see PR #5152 for details.

**Comment by bl-ue** [comment]
cc @julienc91

**Review by bl-ue — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by angauber** [comment]
Thanks everyone :rocket:

**Pull request merged**